### PR TITLE
Document the risks of running a migration without a recycle

### PR DESCRIPTION
**Pull request by eileen-nava**
Following up on [this conversation](https://gsa-tts.slack.com/archives/C0NGESUN5/p1721234296802419) in slack, I wanted to better document the risks of running a migration without a recycle.  Please let me know if there are additional risks that I did not document.  I mostly transcribed the information I learned via the linked slack exchange, which means this section may not be exhaustive.

### CODE REVIEW

File: _articles/appdev-deploy.md [hunk 1]
```diff
@@ -337,3 +338,10 @@ aws-vault exec prod-power -- ./bin/asg-recycle prod idp --skip-migration
 ```bash
 aws-vault exec prod-power -- ./bin/scale-remove-old-instances prod ALL
 ```
+
+### Potential Drawbacks of a No-Migration Recycle
```
Comment (aduth) [inline _articles/appdev-deploy.md:342]: This feels like it'd be a subheading of the same section.

```suggestion
#### Potential Drawbacks of a No-Migration Recycle
```

**Review by aduth — APPROVED** [review]
LGTM!

**Review by gina-yamada — APPROVED** [review]

File: _articles/appdev-deploy.md (hunk 1 above)
Comment (zachmargolis) [reply _articles/appdev-deploy.md:342]: Yes to the subheading also maybe I'd highly as "Risks"?
```suggestion
#### Risks with No-Migration Recycles
```

**Comment by eileen-nava** [comment]
I am having a hard time getting signed commits set up on my local machine.  Once I have signed commits set up, I will reopen a PR for the update and re-request review.

**Pull request closed without merging**